tests/containers/rootless_podman.pm

```diff
@@ -47,6 +47,15 @@ sub run {
         assert_script_run "usermod -a -G systemd-journal $testapi::username";
     }
 
+    if (get_var('TDUP')) {
```

Comment:
```bash
$ grep TDUP variables.md; echo $?
1
```

Reply:
I am not introducing a new variable, just using an existing one. It can be documented in a different PR. Let's not block tests, please

Reply:
I'm with Martin, let's move this forward this was there already before.

Reply:
Of course, just, I don't know what `TDUP` means…

Reply:
Same as `ZDUP`, just with transactional-update.

```
ZDUP | boolean | false | Prescribes zypper dup scenario.
```

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17712